server/master/src/main/java/org/apache/accumulo/master/tableOps/tableImport/ImportTable.java

```diff
@@ -53,16 +54,17 @@
 public class ImportTable extends MasterRepo {
   private static final Logger log = LoggerFactory.getLogger(ImportTable.class);
 
-  private static final long serialVersionUID = 1L;
+  private static final long serialVersionUID = 2L;
```

Comment:
Since the Fate serialization has changed, it will be very important that there are no outstanding import table fate operations outstanding before upgrading. I suspect this is unlikely for most people, but may be something to call out in the release notes.

Reply:
I think we check before upgrade if there are any outstanding FATE ops and if so upgrade will not proceed.  So usually do not need to worry about FATE persistence across versions.